.github/workflows/ci.yml

```diff
@@ -65,8 +65,8 @@ jobs:
     strategy:
       matrix:
         python:
-        - "3.8"  # oldest Python supported by PSF
-        - "3.12"  # newest Python that is stable
+        - "3.9"  # oldest Python supported by PSF
```

Comment:
We should test on the oldest version we support.

Reply:
```suggestion
        - "3.8"  # oldest Python supported by validate-pyproject
```